README.md

```diff
@@ -1 +1,41 @@
-# MeasurementConv
\ No newline at end of file
+# Measurement Conversion Library
+
```

Comment:
I like the name. It tells exactly what the library does.

Reply:
It's what you named it :)